src/app/components/LimitIE9Comment/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+/*
+ * These, when used together, stop anything within them from loading on IE9 and below.
+ */
+
+export const IE9LimitOpen = () => (
+  <div dangerouslySetInnerHTML={{ __html: '<!--[if !IE]> -->' }} /> // eslint-disable-line react/no-danger
```

Comment:
```suggestion
  <div dangerouslySetInnerHTML={{ __html: '<!--[if !IE]><!-->' }} /> // eslint-disable-line react/no-danger
```

Reply:
Hmm this doesn't match `<!--[if !IE]> -->` on https://www.quirksmode.org/css/condcom.html

Reply:
I was referring to https://css-tricks.com/how-to-create-an-ie-only-stylesheet/#article-header-id-3 

Seems like the version you used is a special syntax for down-level conditional comments: https://www.sitepoint.com/internet-explorer-conditional-comments/

In which case, you'll still need to remove the space, but not necessarily add the `<!`

Reply:
Ahh fair, I have more faith in css-tricks :P